FreeRTOS/FreeRTOS-Plus-TCP
# Fix unit test build issues and test failures for IP Utils

Description
-----------
This PR fixes unit test build issues and test failures for IP utils unit tests which were failing after the latest IPv6 integration changes.

Test Steps
-----------
```
cmake -S test/unit-test -B test/unit-test/build/ 
make -C test/unit-test/build/ all 
cd test/unit-test/build/
ctest -E system --output-on-failure --verbose
```

Related Issue
-----------
IP utils unit test cases were failing due to the changes in the source code as part of the IPv6 integration.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
